aws/tagsSSM.go

```diff
@@ -116,3 +117,21 @@ func tagIgnoredSSM(t *ssm.Tag) bool {
 	}
 	return false
 }
+
+func saveTagsSSM(conn *ssm.SSM, d *schema.ResourceData, id, resourceType string) error {
```

Comment:
Nitpick - the name suggests that we're _saving_ tags (in the sense that we're changing them), how would you feel about changing it to something like `readTagsSSM` or anything else that makes it more obvious that this is read-only operation?

Reply:
I agree. I used the name `saveTagsSSM` as that seems to be the convention elsewhere, e.g `saveTagsRDS` in https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/tagsRDS.go#L104. And I suppose it is mutating the passed in `schema.ResourceData`.

So perhaps I could leave it as is for consistency, then propose the name change to all 6 examples in a subsequent PR?

Reply:
> So perhaps I could leave it as is for consistency, then propose the name change to all 6 examples in a subsequent PR?

👍 Sounds good to me!